samples/plain.yaml

```diff
@@ -0,0 +1,31 @@
+kind: DevWorkspace
```

Comment:
with the first try this sample, I got 
```
workspace4aba76e40f844658-678777584-6v79c   0/1     Init:OOMKilled   1          29s
```
But then it got better
```
workspace4aba76e40f844658-678777584-6v79c   1/1     Running   0          69s
```
I used RHPDS

Reply:
Should be fixed by bumping the memory limit to `300Mi`, but it's strange that it would require _so_ much memory.

Reply:
I wonder if bigger projects would require even more memory, which means users need to configure project clone memory limit with devfile attributes or some another way. But I'm OK to postpone it into a dedicated issue https://github.com/devfile/devworkspace-operator/issues/397